### PR TITLE
Minor fix Logger/RDefault/Debug

### PR DESCRIPTION
**Pull request by marios88**
https://github.com/gabordemooij/redbean/issues/663

### CODE REVIEW

File: RedBeanPHP/Logger/RDefault/Debug.php [hunk 1]
```diff
@@ -180,13 +202,13 @@ public function log()
...
-			$bindings = array();
+		    $bindings = NULL;
 		} else {
-			$bindings = func_get_arg( 1 );
+		    $bindings = func_get_arg( 1 );
 		}
 
-		if ( !is_array( $bindings ) ) {
-			return $this->output( $sql );
+		if ( !(boolean) $bindings || !is_array( $bindings )) {
```
Comment (Lynesth) [inline RedBeanPHP/Logger/RDefault/Debug.php]: This is a bit confusing. Something like this seems more understandable in my opinion:
```php
if ( !is_array( $bindings ) || empty( $bindings ) )
```

File: RedBeanPHP/Logger/RDefault/Debug.php [hunk 2]
```diff
@@ -180,13 +202,13 @@ public function log()
 		$sql = func_get_arg( 0 );
 
 		if ( func_num_args() < 2) {
-			$bindings = array();
+		    $bindings = NULL;
```
Comment (Lynesth) [inline RedBeanPHP/Logger/RDefault/Debug.php]: Tabs, not spaces

File: RedBeanPHP/Logger/RDefault/Debug.php [hunk 3]
```diff
@@ -180,13 +202,13 @@ public function log()
...
 		} else {
-			$bindings = func_get_arg( 1 );
+		    $bindings = func_get_arg( 1 );
```
Comment (Lynesth) [inline RedBeanPHP/Logger/RDefault/Debug.php]: tabs, not spaces

File: RedBeanPHP/Logger/RDefault/Debug.php [hunk 4]
```diff
@@ -180,13 +202,13 @@ public function log()
...
 		}
 
-		if ( !is_array( $bindings ) ) {
-			return $this->output( $sql );
+		if ( !(boolean) $bindings || !is_array( $bindings )) {
+		    return $this->output( $sql);
```
Comment (Lynesth) [inline RedBeanPHP/Logger/RDefault/Debug.php]: tabs, not spaces

**Comment by gabordemooij** [comment]
wow what a lot of changes...

**Pull request closed without merging**